pkg/apis/kudo/v1beta1/instance_types.go

```diff
@@ -78,22 +78,55 @@ type AggregatedStatus struct {
 type PlanStatus struct {
 	Name            string                `json:"name,omitempty"`
 	Status          ExecutionStatus       `json:"status,omitempty"`
+	Message         string                `json:"message,omitempty"`
```

Comment:
just thinking out loud - wdyt about:
```suggestion
	ErrorMessage         string                `json:"errorMessage,omitempty"`
```
`Message` sounds somewhat generic to me.

Reply:
That was my intention. I was hoping that we could even put some success messages there in the future if makes sense. also it felt aligned with how core types for k8s look like

Reply:
:+1: to Message.